### PR TITLE
Test markdown-input and improve the toggle button

### PR DESCRIPTION
**Pull request by txsmith**
## screenshot from 2016-05-24 14 30 28

screenshot from 2016-05-24 14 30 42

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -20,6 +20,8 @@
   <body>
     <script>
       WCT.loadSuites([
+        'util/markdown-input.html',
+        'util/markdown-input.html?dom=shadow',
         'ajax/authenticated-ajax.html',
         'ajax/authenticated-ajax.html?dom=shadow'
       ]);
```
Comment (eanker) [inline test/index.html:27]: This is no alphabetic order, so please change the order (ajax, util)

File: src/util/markdown-input.html [hunk 2]
```diff
@@ -65,16 +73,27 @@
...
         }
       },
-      _hidePreview: function(previewToggle, disabled) {
+      _getHidePreview: function(previewToggle, disabled) {
         var shouldBeHidden = !disabled && !previewToggle;
         this.toggleClass('noPreview', shouldBeHidden);
         return shouldBeHidden;
+      },
+      _getToggleText: function(active) {
+        if (active) {
```
Comment (TimvdLippe) [inline src/util/markdown-input.html:92]: I would collapse this to just `return active ? 'Edit' : 'Preview';`

Comment (txsmith) [inline src/util/markdown-input.html:92]: I personally think inline if-statements should be avoided if they do not provide a significant benefit. In this case a full if statement is equivalent in functionality and easier to read.

**Comment by TimvdLippe** [comment]
Confirmed working, nice!

**Pull request merged**